omnibus/config/templates/datadog-agent/sysvinit_debian.trace.erb

```diff
@@ -42,9 +42,9 @@ do_start()
 	#   0 if daemon has been started
 	#   1 if daemon was already running
 	#   2 if daemon could not be started
-	start-stop-daemon --start --quiet --pidfile $PIDFILE --exec $AGENTPATH --test > /dev/null \
+	start-stop-daemon --start --test --quiet --pidfile $PIDFILE --start-as $AGENTPATH > /dev/null \
```

Comment:
`--start-as` vs `--startas` in the other files.

Reply:
Homogeneized as `--startas`